extensions/amp-form/0.1/amp-form.js

```diff
@@ -242,27 +247,32 @@ export class AmpForm {
   installEventHandlers_() {
     this.form_.addEventListener(
         'submit', this.handleSubmitEvent_.bind(this), true);
+
     this.form_.addEventListener('blur', e => {
       checkUserValidityAfterInteraction_(dev().assertElement(e.target));
       this.validator_.onBlur(e);
     }, true);
+
+    const cleanup = () => {
```

Comment:
`cleanup` too generic given this methods deals with non-verifier code as well. Maybe `afterVerifierCommit` ?